app/views/admin/users/show.html.erb

```diff
@@ -9,3 +9,5 @@
   <li><b>Small business</b>: <%= @view_model.small_business %></li>
   <li><b>Contracting officer</b>: <%= @view_model.contracting_officer %></li>
 </ul>
+
+<%= render partial: @view_model.bids_partial %>
```

Comment:
oh wait I see it uses `@view_model` - this is super minor but I usually try to avoid instance vars in view partials (only use vars that are passed) - the benefit of that being that you only need to go to where something is rendered to know where the ivar comes from.